### PR TITLE
OESS handler integration

### PR DESCRIPTION
**Pull request by ibaldin**
This add handler integration for OESS (similar to OSCARS) via a new RENCI-NRIG/na2-oess-plugin plugin to NA2. 

Also adds a new RDF file with new form of URNs for OESS @mcevik0 

Note ScriptExecutor had to be modified to a threaded version - the old version randomly stopped working.

### CODE REVIEW

**Comment by ibaldin** [comment]
One possible area of concern - this will be merged on top of Java 8.0 changes? Should work but hasn't been tested and it would mean we need deploy the Internet2/ESnet AM into Java 8 environment.

File: handlers/oess/src/main/java/orca/handlers/network/oess/tasks/UrnRemapperTask.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import org.apache.tools.ant.BuildException;
+
+/**
+ * Remap URNs based on a property file
+ * For now the property file has one property with an even number
+ * of comma-separated URNs. The odd-ones are 'from', the even ones are 'to'
+ * @author ibaldin
+ *
+ */
+public class UrnRemapperTask extends OrcaAntTask {
```
Comment (hinchliff) [inline handlers/oess/src/main/java/orca/handlers/network/oess/tasks/UrnRemapperTask.java]: Can we use the Oscars [UrnRemapperTask](https://github.com/RENCI-NRIG/orca5/blob/orca-5.1.6/handlers/oscars/src/main/java/orca/handlers/network/oscars/tasks/UrnRemapperTask.java) without duplicating the code?

File: core/util/src/main/java/orca/util/ScriptExecutor.java [hunk 2]
```diff
@@ -65,6 +102,12 @@ public ExecutionResult execute() throws Exception
             proc = rt.exec(cmdArray);
         }
 
+        ReadStream stdinRstr = new ReadStream("stdin", proc.getInputStream(), logger);
+        ReadStream errRstr = new ReadStream("stderr", proc.getErrorStream(), logger);
+        
+        stdinRstr.start();
+        errRstr.start();
+        /*
```
Comment (hinchliff) [inline core/util/src/main/java/orca/util/ScriptExecutor.java]: We should delete commented out code.

**Comment by ibaldin** [comment]
My thinking was we should eventually stop compiling in oscars plugin into the code. But I still want the URN Remapper task. Most likely it needs a different home.

File: core/util/src/main/java/orca/util/ScriptExecutor.java (hunk 2 above)
Comment (ibaldin) [inline core/util/src/main/java/orca/util/ScriptExecutor.java]: OK I'll delete. I noticed other implementations of the same executor. Seems like maybe this has come up before. I don't know why this code stopped working or why the new one works. They are mostly equivalent, except new code is multi-threaded.

Comment (ibaldin) [inline core/util/src/main/java/orca/util/ScriptExecutor.java]: This is done now.

**Comment by ibaldin** [comment]
UrnMapper has been moved to common tasks. Removed from OESS package

**Pull request merged**